src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/globalnav/GlobalNavigation.java

```diff
@@ -9,6 +9,7 @@
 import com.wikia.webdriver.pageobjectsfactory.pageobject.SearchPageObject;
 
 import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.openqa.selenium.NoSuchElementException'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)